File: chroma-league/src/test/java/com/bonepl/chromaleague/hud/animations/AllyChemtechDragonKillAnimationTest.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class AllyChemtechDragonKillAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playAllyChemtechDragonKillAnimation() {
        new AnimationTester().testAnimation(new AllyChemtechDragonKillAnimation());
    }
}

File: test/hud/animations/AllyChemtechDragonKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {AllyChemtechDragonKillAnimation} from '../../../src/hud/animations/AllyChemtechDragonKillAnimation.js';

/**
 * Integration test for ally chemtech dragon kill animation on actual hardware.
 */
describe('AllyChemtechDragonKillAnimation Integration', () => {
    test('playAllyChemtechDragonKillAnimation', async () => {
        await new AnimationTester().testAnimation(new AllyChemtechDragonKillAnimation());
    }, 30000);
});
